### PR TITLE
fix typos and add redirect url

### PR DESCRIPTION
**Pull request by finp**
## Motivation
tidy up text displayed by installer

## What
correct grammar and use redirect urls rather than hard coding for self signed certs docs

### CODE REVIEW

File: installer/install.sh [hunk 1]
```diff
@@ -332,7 +332,7 @@ function check_oc_cluster_up() {
 # To read and check docker credentials
 function read_docker_hub_credentials() {
   info_msg "The Mobile installer requires valid DockerHub credentials to communicate with the DockerHub API."
-  info_msg "If you enter with an invalid credentials then Mobile Services will not be available in the Service Catalog."
+  info_msg "If you enter invalid credentials then Mobile Services will not be available in the Service Catalog."
```
[the remark this answers is not in the document]
Comment (camilamacedo86) [reply installer/install.sh]: It is true, it is not possible to process without valid credentials.

File: installer/install.sh [hunk 2]
```diff
@@ -456,7 +456,7 @@ function run_ansible_tasks() {
   fi
   install_with_success_msg "Mobile Services"
   info_msg "See the Mobile Services in your OpenShift Console. URL: https://${DEFAULT_CLUSTER_IP}:8443/console/"
-  info_msg "For information on how to enable TLS communication on your device to this cluster see https://docs.aerogear.org/aerogear/latest/getting-started.html#using-self-signed-certificates-in-mobile-apps"
+  info_msg "For information on how to enable TLS communication on your device to this cluster see https://docs.aerogear.org/external/installer/self-signed-cert.html"
```
Comment (camilamacedo86) [inline installer/install.sh]: In the PR https://github.com/aerogear/mobile-core/pull/206 I added this change

**Comment by camilamacedo86** [comment]
@finp could you rebase it with the master?

**Comment by finp** [comment]
@camilamacedo86  done

**Pull request merged**